packages/devextreme/js/__internal/grids/grid_core/editing/m_editing.ts

```diff
@@ -1770,8 +1764,7 @@ class EditingControllerImpl extends modules.ViewController {
     return deferred.promise();
   }
 
-  // @ts-expect-error
-  private _resolveAfterSave(deferred, { cancel, error } = {}) {
+  private _resolveAfterSave(deferred, { cancel = false, error = undefined } = {}) {
```

Comment:
```suggestion
  private _resolveAfterSave(deferred, { cancel = undefined, error = undefined } = {}) {
```
Minor
Let's not change behavior, looks unrelated to PR

Reply:
done